### PR TITLE
[IA-4103] Fix spark cost calculation

### PR DESCRIPTION
**Pull request by lucymcnatt**

### CODE REVIEW

**Review by jdcanas — APPROVED** [review]
Ah thanks, guess our tests weren't fully comprehensive in this case (or it was always a bug?). Seems like its worth having a test that exercises this code path for cost...

**Review by sjacobs146 — CHANGES_REQUESTED** [review]
It appears that we are still seeing "Unknown" for Hail Image before selecting Spark Cluster option and selecting Default image and compute type of Spark single node.

File: src/pages/workspaces/workspace/analysis/utils/cost-utils.ts [hunk 1]
```diff
@@ -60,8 +60,8 @@ export const runtimeConfigBaseCost = (config: GoogleRuntimeConfig): number => {
   const computeRegion = getNormalizedComputeRegion(config)
 
   const costForDataproc: number = isDataprocConfig(config) ?
-    config.masterDiskSize + config.numberOfWorkers *
-      getDefaultIfUndefined(config.workerDiskSize, defaultDataprocWorkerDiskSize) *
+    (config.masterDiskSize + config.numberOfWorkers *
+      getDefaultIfUndefined(config.workerDiskSize, defaultDataprocWorkerDiskSize)) *
```
Comment (mlilynolting) [inline src/pages/workspaces/workspace/analysis/utils/cost-utils.ts:64]: wow

**Review by mlilynolting — APPROVED** [review]
LGTM - I think Susan's requested changes have been made; I can't recreate the buggily high prices for Spark clusters or the Unknown prices for Spark single node. ⭐️

**Review by sjacobs146 — APPROVED** [review]

**Comment by lucymcnatt** [comment]
I can't approve cuz I opened it, but LGTM also

**Pull request merged**